src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -24,7 +24,7 @@ export const EditProblemView = ({
 
   return (
     <EditorContainer getContent={getContent}>
-      <Container fluid>
+      <Container fluid className="mt-3 pl-4 pr-4">
```

Comment:
```suggestion
      <Container fluid className="mt-3 py-4">
```

Reply:
shouldn't it be px-4 if it's left and right? Confused

Reply:
yes. sorry i got x and y confused